plugins/services/src/js/structs/__tests__/Framework-test.js

```diff
@@ -283,4 +283,20 @@ describe("Framework", function() {
       });
     });
   });
+
+  describe("getWebURL", () => {
+    // this test mostly makes sure that util isnt removed from `Service` struct - see ServiceUtil tests for more details
+    it("returns url if service is SDK Service providing DCOS_SERVICE_WEB_PATH label", function() {
+      const service = new Framework({
+        labels: {
+          DCOS_SERVICE_NAME: "foo",
+          DCOS_SERVICE_PORT_INDEX: "80",
+          DCOS_SERVICE_SCHEME: "https",
+          DCOS_COMMONS_API_VERSION: "notnull",
+          DCOS_SERVICE_WEB_PATH: "/bar"
```

Comment:
are we sure that there's always a slash in front? can i look that up somewhere?

Reply:
well… we _should_… but I think adding a quick catch for that is actually a good idea… will do 👍

Reply:
this would also make the logic in getWebURL a little easier! 🎉